cmake/Modules/FindMagic_EP.cmake

```diff
@@ -90,15 +90,15 @@ if(NOT TILEDB_LIBMAGIC_EP_BUILT)
     ExternalProject_Add(ep_magic
       PREFIX "externals"
       GIT_REPOSITORY "https://github.com/TileDB-Inc/file-windows.git"
-      GIT_TAG "5a53ee05e3ee71424953196171d994969b916c51"
+      GIT_TAG "5.38.1.tiledb"
       GIT_SUBMODULES_RECURSE TRUE
       UPDATE_COMMAND ""
       CMAKE_ARGS
         -DCMAKE_INSTALL_PREFIX=${TILEDB_EP_INSTALL_PREFIX}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
         "-DCMAKE_C_FLAGS=${CFLAGS_DEF}"
         -Dlibmagic_STATIC_LIB=ON
-        -DBUILD_HOST_TOOLS_MACOS_UNIVERSAL=ON
+        -DBUILD_MAGIC_MACOS_UNIVERSAL=$ENV{BUILD_MAGIC_MACOS_UNIVERSAL}
```

Comment:
Works ok when BUILD_MAGIC_MACOS_UNIVERSAL is not defined?

Reply:
It's not defined in the other CI jobs, which passed